lms/static/certificates/sass/_components.scss

```diff
@@ -0,0 +1,641 @@
+// ------------------------------
+// // Open edX Certificates: Components
+
+// About: styling for specific UI components ranging from global to modular.
+
+// #OVERRIDE - COPY
+// #OVERRIDE - HEADINGS
+// #OVERRIDE - BUTTONS
+
+// #DEPTH
+// #MESSAGES
+// #IMAGES
+
+// #HEADER APP
+// #BANNER
+// #INTRODUCTION
+// #ACCOMPLISHMENT
+// #RENDERING
+// #ACCOMPLISHMENT RELATED
+// #CERTIFICATE
+// #ABOUT
+
+// ------------------------------
+// #OVERRIDE - COPY
+// ------------------------------
+// for some reason in Sass compilation, function calls as variables aren't playing well :/
```

Comment:
NOTE: @clrux  and I need to investigate why these Sass function calls can't be passed in as variables at this level. Light redundancy is needed for now with that in mind.